front/src/components/Artwork.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div class="artwork">
+    <div class="top">
+      <a class="title info">{{ artwork.name }}</a>
+
+      <img alt="Artwork Image" src="../assets/test.jpeg" class="image" />
+    </div>
+
+    <div class="description">
+      <p
+        v-if="artwork.description && artwork.description[$i18n.locale]"
+        class="desc"
+      >
+        {{ artwork.description[$i18n.locale] }}
+      </p>
+
+      <p class="location">
+        <a class="info">{{ $t("Artwork.location") }}:</a>
+        {{ artwork.location.name }}
+      </p>
+
+      <p class="author">
+        <a class="info">{{ $t("Artwork.author") }}: </a>
+        <a v-for="(artist, index) in artwork.artists" :key="index"
+          >{{ artist }}<b v-if="index != artwork.artists.length - 1">, </b>
+        </a>
+      </p>
+
+      <p class="period">
+        <a class="info">{{ $t("Artwork.period") }}:</a>
+        {{ artwork.creationPeriod.minDate
+        }}<a v-if="artwork.creationPeriod.maxDate"
+          >-{{ artwork.creationPeriod.maxDate }}</a
+        >
+      </p>
+
+      <a v-if="artwork.wikiLink" :href="artwork.wikiLink" target="_blank">{{
+        $t("Artwork.wiki")
+      }}</a>
+    </div>
+  </div>
+</template>
+
+<script>
+import artworkData_list from "../assets/V0-list.json";
+
+export default {
+  name: "ArtWork",
+  props: {
+    id: {
+      type: Number,
```

Comment:
I got the following warning
`Invalid prop: type check failed for prop "id". Expected Number with value 1, got String with value "1". `
![image](https://github.com/Museum-GO/MuseumGO-App/assets/59999324/abb79ab2-c589-484c-96de-c77be10cdcd6)
Please fix